otx/algorithms/classification/configs/configuration.yaml

```diff
@@ -33,7 +33,7 @@ learning_parameters:
       make it unstable.
     editable: true
     header: Learning rate
-    max_value: 0.1
+    max_value: 1.0
```

Comment:
Doesn't it affect the training lr scheduler not only for self-sl but other tasks?

Reply:
That was my bad. I realized it just max value for ranging learning rate.

Reply:
As we use lr scheduler to reduce lr, max_value might be fine, but it is needed to check lr scheduler algorithm in the mpa.

Reply:
Since this is just `max_value`. it couldn't affect to others